bigquery/samples/tests/test_list_tables.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .. import list_tables
+
+
+def test_list_tables(capsys, client, dataset_id, table_id):
+
+    list_tables.list_tables(client, dataset_id)
+    out, err = capsys.readouterr()
+    assert "Tables contained in '{}':".format(dataset_id) in out
+    assert "{}".format(table_id) in out
```

Comment:
Since `table_id` is already a string, no need for `"{}".format(`. This can be `assert table_id in out`.

Reply:
Updated!